dynamometer-workload/src/main/java/com/linkedin/dynamometer/workloadgenerator/audit/AuditLogHiveTableParser.java

```diff
@@ -25,7 +25,7 @@
  *   INSERT OVERWRITE DIRECTORY '${outputPath}'
  *   SELECT (timestamp - ${startTimestamp} AS relativeTimestamp, ugi, cmd, src, dst, ip
  *   FROM '${auditLogTableLocation}'
- *   WHERE timestamp >= ${startTimestamp} AND timestamp < ${endTimestamp}
+ *   WHERE timestamp &gt;= ${startTimestamp} AND timestamp &lt; ${endTimestamp}
```

Comment:
This works, but looks ugly in the code. If you wrap the whole thing in an `{@code}` I think it will also work:
```
{@code
<pre>
  ...
</pre>
}
```
Here's a good discussion of the subject: https://reflectoring.io/howto-format-code-snippets-in-javadoc/